docs/FAQ.md

```diff
@@ -520,6 +520,9 @@ If you are getting a warning `Avro library not found by the RAPIDS plugin.` or i
 Spark job by using the `--jars` or `--packages` option followed by the file path or maven path to 
 RAPIDS jar since that is the preferred way to run RAPIDS accelerator. 
 
+NOTE: You could use `--extraClassPath` followed by the path to Avro and the RAPIDS Plugin jar but one
```

Comment:
Is this supposed to be `--extraClassPath` or `--conf spark.driver.extraClassPath=` or `--conf spark.executor.extraClassPath=`.  I've never been able to find a reference to `--extraClassPath` in the Spark online documentation anywhere, but `--jars` and `--packages` is definitely there.

Reply:
correct, the confs are 

spark.driver.extraClassPath
spark.executor.extraClassPath (it's [discouraged](https://spark.apache.org/docs/latest/configuration.html))  but we need it if we don't want to recommend patching SPARK_HOME/jars

In the client mode REPL the conf setting is too late, and therefore there exists `--driver-class-path`

Reply:
>  I've never been able to find a reference to --extraClassPath in the Spark online documentation anywhere

Sorry, that's my fault. I could swear I saw that somewhere but obviously not